### PR TITLE
Added new Currency provider

### PR DESCRIPTION
**Pull request by imrancio**
Yahoo provider for Currency converter does not work anymore, thus only returning few results from European Central Bank. I replaced it with a free API call to currencyconverterapi.com. Also added some usage instructions for the extension.

### CODE REVIEW

**Comment by ManuelSchneid3r** [comment]
bildschirmfoto vom 2018-12-23 18-06-52

works well here

File: Currency.py [hunk 1]
```diff
@@ -50,34 +52,41 @@ def convert(self, amount, src, dst):
             dst_rate = self.exchange_rates[dst]
             return str(amount / src_rate * dst_rate)
 
-class Yahoo:
+class Api:
```
Comment (ManuelSchneid3r) [inline Currency.py]: please give this a reasonable name an reintroduce yahoo.

File: Currency.py [hunk 2]
```diff
@@ -14,6 +15,7 @@
 __iid__ = "PythonInterface/v0.1"
 __prettyname__ = "Currency converter"
 __version__ = "1.0"
+__trigger__ = "cc "
```
Comment (ManuelSchneid3r) [inline Currency.py]: why do you introduce a trigger? actually it was intended to work without the trigger.

File: Currency.py (hunk 1 above)
Comment (imrancio) [reply Currency.py]: Can do. Yahoo does not work for me. I tested the regex on the html that was returned to me and it was redirected to Yahoo AU. So there may be inconsistencies in results across regions and the Yahoo Finance API has been deprecated for a while.
screenshot_2018-12-24_16-48-30

**Comment by imrancio** [comment]
screenshot_2018-12-24_17-34-34
It works, but only with proper currency codes. Yahoo would have been nice for more general search.

**Comment by imrancio** [comment]
> bildschirmfoto vom 2018-12-23 18-06-52
> 
> works well here

Also I don't know if we're looking at the same currency, but your result seems way off.

**Comment by imrancio** [comment]
I guess this will never be merged in. At least, the other 2 currency providers have been fixed now.

**Pull request closed without merging**